features/cellular/framework/targets/QUECTEL/BC95/QUECTEL_BC95.cpp

```diff
@@ -78,3 +75,12 @@ nsapi_error_t QUECTEL_BC95::init()
 
     return _at->unlock_return_error();
 }
+
+nsapi_error_t QUECTEL_BC95::reset()
+{
+    _at->lock();
```

Comment:
should this also reset state machine like bsae class does?

Reply:
yes, need to add AT_CellularDevice::shutdown();